### ISSUE TITLE
directive for curl client cmd, http server resp

### ISSUE THREAD

**Issue by kbhawkey** [russell/fairy-slipper#35]
Some of the service method descriptions embed curl commands along with http responses.  Would it be useful to pull these out of the description and create a directive to tag as, for example, http req/resp output.  This text could be associated with a curl/http (cmd, code block) widget and used within the display, outside of the description.

**Comment by annegentle** [comment]
Yes, I think it would be useful -- preserves the curl commands while also lets us use the examples eventually in the display. Good thinking.
